bionty/base/_public_ontology.py

```diff
@@ -108,6 +108,26 @@ def __init__(
             except AttributeError:
                 pass
 
+    def _validate_param(self, param_name: str, value: str | None) -> None:
```

Comment:
If this equates type annotation checks, why not used `typechecked` or pydantic?

I assume it's because there is more going on.

Small note re naming: `param` in lamindb has a special meaning akin to `feature`. When we talk about plain Python function args, we say `args` aligned with Google-style docstrings.

Reply:
(Should at some point update the "Paramters" header and replace it with "Arguments".